### PR TITLE
GLT-620 update save button placement on Handsontable pages

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

**Comment by apmasell** [comment]
:+1:

File: miso-web/src/main/webapp/pages/editSample.jsp [hunk 1]
```diff
@@ -739,16 +749,7 @@
...
-      <c:choose>
-      <c:when test="${detailedSample}">
-        <button id="saveDetailed" class="disabled" disabled="disabled" onclick="Sample.hot.saveDetailedData();">Save</button>
-      </c:when>
-      <c:otherwise>
-        <button id="savePlain" onclick="Sample.hot.savePlainData();">Save Bulk</button>
-      </c:otherwise>
-      </c:choose>
-    </div>
+    <div class="clear"></div>
```
Comment (djcooke) [inline miso-web/src/main/webapp/pages/editSample.jsp]: Is this div still useful for styling or something?

**Comment by djcooke** [comment]
:+1:

File: miso-web/src/main/webapp/pages/editSample.jsp (hunk 1 above)
Comment (callunity) [inline miso-web/src/main/webapp/pages/editSample.jsp]: Here's the rendering without it, just for fun: cleardiv

**Pull request merged**